commands/assign-dish.js

```diff
@@ -0,0 +1,35 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { getMenu, saveMenu } = require('./../menu-memory/menu.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('assign-dish')
+		.setDescription('assign a dish to a user')
+		.addUserOption(option => option.setName('user').setDescription('Who is bringing the dish>'))
+		.addStringOption(option => option.setName('dish').setDescription('Enter a dish')),
+	async execute(interaction) {
+		let user = interaction.options.getUser('user');
+		const dish = interaction.options.getString('dish');
+		let menu = await getMenu();
+		
+		if (dish == undefined) {
+			return interaction.reply(`No dish was provided to sign up for`);
+		}
+		
+		if (menu.Dishes[dish] != undefined && menu.Dishes[dish] != "") {
+			return interaction.reply(`${dish} is already being brought by <@${menu.Dishes[dish]}>`)
```

Comment:
Whats up with your quotes?
`
'
"

Standards suggest only using single quotes for all

Reply:
What is the difference between single and double quotes in js?

Reply:
@GuardianG13 I think the double quote is a copy-paste from
JSON 

please [propose the fix](http://haacked.com/archive/2019/06/03/suggested-changes/) 

@ChinchillaMan99 
When manually building HTML it matters but otherwise it’s just what you do and don’t have to escape.